README.md

```diff
@@ -18,6 +18,10 @@ If you have a suggestion or question, open an [issue](https://github.com/redhat-
 
 See the [Contributing guide](CONTRIBUTING.md) for information on how to contribute an update.
 
+## Administrator responsibilities
+
+See the list of [administrator responsibilities](ADMIN.adoc) for more information on what to do in your role as the administrator of this repository.  
```

Comment:
@bergerhoffer I linked to the admin responsibilities list from README.adoc. Please let me know whether this is OK, or you'd like me to change anything about this section. Thank you.